### PR TITLE
Hunterise with protobuf and pybind11 packages

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

**Comment by xsacha** [comment]
Test was failing with being unable to find Protobuf, so I added a find_package in the Config.

File: CMakeLists.txt [hunk 1]
```diff
@@ -13,8 +13,37 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 cmake_policy(SET CMP0063 NEW)
 
+# Hunter
+include(cmake/Hunter/HunterGate.cmake)
```
Comment (ruslo) [inline CMakeLists.txt:17]: I don't see this module committed.

File: CMakeLists.txt [hunk 2]
```diff
@@ -568,11 +603,12 @@ install(FILES
   ${PROJECT_BINARY_DIR}/ONNXConfig.cmake
   DESTINATION share/cmake/ONNX
   COMPONENT dev)
-install(EXPORT ONNXTargets DESTINATION share/cmake/ONNX)
+install(EXPORT ONNXTargets NAMESPACE "${PROJECT_NAME}::" DESTINATION share/cmake/ONNX)
 install(TARGETS
   onnx onnx_proto
   onnxifi onnxifi_dummy onnxifi_loader
-  EXPORT ONNXTargets DESTINATION lib)
+  EXPORT ONNXTargets
```
Comment (ruslo) [inline CMakeLists.txt]: Try to keep change minimal. There is no need to modify this line.

File: CMakeLists.txt [hunk 3]
```diff
@@ -353,16 +382,22 @@ if(BUILD_ONNX_PYTHON)
...
+    hunter_add_package(pybind11)
+    find_package(pybind11 CONFIG REQUIRED)
+    target_link_libraries(onnx_cpp2py_export PUBLIC pybind11::pybind11)
   else()
-    if(EXISTS ${ONNX_ROOT}/third_party/pybind11/include/pybind11/pybind11.h)
+    find_package(pybind11 2.2)
+    if(pybind11_FOUND)
       target_include_directories(onnx_cpp2py_export PUBLIC
-        ${ONNX_ROOT}/third_party/pybind11/include)
+        ${pybind11_INCLUDE_DIRS})
```
Comment (ruslo) [inline CMakeLists.txt]: We are not entering the `else` block, you can keep the code to minimize number of changes in patch.

**Review by ruslo — CHANGES_REQUESTED** [review]
Please split changes

**Comment by xsacha** [comment]
Ready to go

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/onnx/releases/tag/v1.4.1-p0

**Pull request merged**